benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -106,19 +106,18 @@ public BenchmarkExecutionResult execute(Benchmark benchmark, int benchmarkOrdina
 
     private BenchmarkExecutionResult executeBenchmark(Benchmark benchmark, Optional<ZonedDateTime> executionTimeLimit)
     {
-        BenchmarkExecutionResultBuilder resultBuilder = new BenchmarkExecutionResultBuilder(benchmark);
-        List<QueryExecutionResult> executions;
-        try {
-            executions = executeQueries(benchmark, benchmark.getPrewarmRuns(), true, executionTimeLimit);
```

Comment:
Can't `executeQueries` throw an exception? You moved it outside of `try` clause.

This looks like unrelated change. Undo or separate commit?

Reply:
I moved it out, because there are listeners that rely on the result being started, which only happens after pre-warm queries are finished. These exceptions cover the original one, which makes it very hard to debug benchmark configs. This is easily reproducible when you mess up the variables and try to run an invalid query. Moving pre-warm queries outside of the try block makes the whole driver fail fast with an actionable exception. I think in this case we don't want to continue the benchmark, if the whole benchmark is invalid.

Reply:
> Moving pre-warm queries outside of the try block makes the whole driver fail fast with an actionable exception. I think in this case we don't want to continue the benchmark, if the whole benchmark is invalid.

Pre-warm failure should only fail that particular query being benchmarked and not the whole benchto run.

Reply:
I reverted this change, we can figure this out later.